src/Navigation/Navigation.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { NavigationContainer } from '@react-navigation/native';
+import { AsyncStorage } from 'react-native';
```

Comment:
It seems you are not using `AyncStorage` in this file

Reply:
👍